templates/inspec/singular_resource.erb

```diff
@@ -67,7 +67,7 @@ class <%= object.name -%> < GcpResourceBase
 
   def parse
 <%
-  parse_code = object.all_user_properties.map do |prop|
+  parse_code = object.all_user_properties.reject(&:exclude_reader).map do |prop|
```

Comment:
Why wouldn't you just exclude the property altogether?

Reply:
Good catch! I tried that at one point, but had a bug and forgot to come back to it after I fixed it.

Removed the override and excluded the `name` field